lib/postcode_validation/use_case/address_list.rb

```diff
@@ -22,7 +22,7 @@ def execute(postcode:, country:)
 
       def formatted(addresses)
         addresses.map do |address|
-          { street_address: address.street_address, place: address.place }
+          { street_address: address.street_address }
```

Comment:
I compacted place and street_address into one string, but after you mentioned this, I can see value in keeping them separate.  So I separated them again.